flutter/shell/platform/tizen/channels/platform_channel.h

```diff
@@ -22,21 +25,37 @@ class PlatformChannel {
   virtual ~PlatformChannel();
 
  private:
+  using ClipboardCallback = std::function<void(const std::string& data)>;
+
   void HandleMethodCall(
       const MethodCall<rapidjson::Document>& call,
       std::unique_ptr<MethodResult<rapidjson::Document>> result);
 
   void SystemNavigatorPop();
   void PlaySystemSound(const std::string& sound_type);
   void HapticFeedbackVibrate(const std::string& feedback_type);
+  void GetClipboardData(ClipboardCallback on_data);
+  void SetClipboardData(const std::string& data);
+  bool ClipboardHasStrings();
   void RestoreSystemUiOverlays();
   void SetEnabledSystemUiOverlays(const std::vector<std::string>& overlays);
   void SetPreferredOrientations(const std::vector<std::string>& orientations);
 
   std::unique_ptr<MethodChannel<rapidjson::Document>> channel_;
 
   // A reference to the native view managed by FlutterTizenView.
-  TizenViewBase* view_;
+  TizenViewBase* view_ = nullptr;
+
+  // The clipboard history manager.
+  [[maybe_unused]] cbhm_h cbhm_handle_ = nullptr;
+
+  // A container that holds clipboard data during the engine lifetime.
+  //
+  // Only used by profiles that do not support the Tizen clipboard API
+  // (wearable and TV).
+  std::string clipboard_;
```

Comment:
Actually I was trying to minimize the use of `#ifdef` (`#if`) as much as possible because it usually makes code less readable but if you want the code to be more explicit and clear, I'll fix as you suggested.